### PR TITLE
HSEARCH-4135 + HSEARCH-4136 Use hibernate.search.backend.uris and upgrade to ES 7.8 in Jenkinsfile of backend performance tests

### PR DESCRIPTION
**Pull request by yrodiere**
* [HSEARCH-4136](https://hibernate.atlassian.net/browse/HSEARCH-4136): Use hibernate.search.backend.uris rather than hibernate.search.backend.hosts for tests in POMs
* [HSEARCH-4135](https://hibernate.atlassian.net/browse/HSEARCH-4135): Upgrade to ES 7.8 in Jenkinsfile of backend performance tests

### CODE REVIEW

**Comment by yrodiere** [comment]
Full build: https://ci.hibernate.org/job/hibernate-search/job/PR-2467/3/

**Comment by yrodiere** [comment]
The Jenkinsfile for performance tests works again: https://ci.hibernate.org/view/Search/job/hibernate-search-performance/job/backend-elasticsearch/50/console

File: documentation/src/test/resources/META-INF/persistence.xml [hunk 1]
```diff
@@ -54,35 +43,6 @@
...
-            <property name="hibernate.search.backend.aws.region"
-                      value="${test.elasticsearch.connection.aws.region}"/>
-            <property name="hibernate.search.backend.aws.credentials.type"
-                      value="${test.elasticsearch.connection.aws.credentials.type}"/>
-            <property name="hibernate.search.backend.aws.credentials.access_key_id"
-                      value="${test.elasticsearch.connection.aws.credentials.access_key_id}"/>
-            <property name="hibernate.search.backend.aws.credentials.secret_access_key"
-                      value="${test.elasticsearch.connection.aws.credentials.secret_access_key}"/>
-            <property name="hibernate.search.automatic_indexing.synchronization.strategy"
-                      value="sync"/>
```
Comment (fax4ever) [inline documentation/src/test/resources/META-INF/persistence.xml:85]: Removing the update of these properties can be an issue for the correct execution of `GettingStartedDefaultAnalysisIT` test.

Comment (fax4ever) [reply documentation/src/test/resources/META-INF/persistence.xml:85]: I think that the issue is related to the fact that host+port takes the precedence over the uris...

**Review by fax4ever — COMMENTED** [review]
Just one comment.
+1 to merge as soon as it is fixed

File: documentation/src/test/resources/META-INF/persistence.xml (hunk 1 above)
Comment (yrodiere) [reply documentation/src/test/resources/META-INF/persistence.xml:85]: Yes, I'll look into it.

**Comment by yrodiere** [comment]
Merged, thanks

**Pull request merged**